src/components/VSelect/VSelect.vue

```diff
@@ -329,7 +329,21 @@
           click: () => {
             if (!this.isActive) this.isActive = true
           },
-          keydown: this.onKeyDown // Located in mixins/autocomplete.js
+          keydown: (e) => {
+            if (!this.isActive &&
+                !this.multiple &&
```

Comment:
Potentially from the one selected furthest down in the DOM. I'm thinking of some other things I can do to improve this, and may take it a different route.